### PR TITLE
Fix check error

### PR DESCRIPTION
**Pull request by jrowen**
I made a few minor updates to fix a check error, enable testing on Windows and re-organize the LICENSE files with the hope of making a possible future CRAN submission easier.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,2 @@
...
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+YEAR: 2015
+COPYRIGHT HOLDER: Continuum Analytics, Inc., Kenton Russell, and Ryan Hafen
```
Comment (hafen) [inline LICENSE]: I'm not sure what to do here.  I don't know if we claim copyright for every author - I've heard of in open source projects assigning copyright to a single entity to make it easy to maintain / enforce.  Also there are sometimes employer issues with things like this so good to check with @timelyportfolio.

**Comment by hafen** [comment]
Excellent.  Thanks!  Looks great - just one comment inline.

**Comment by hafen** [comment]
By the way, speaking of CRAN, I should be able to start knocking out some issues / features soon.  I'll put up a CRAN milestone soon.

**Comment by jrowen** [comment]
I just went through this with `rhandsontable` and applied a similar approach here, which tries to mirror the approach used in the htmlwidgets created by rstudio and inudes the feedback I received from CRAN.  I think you probably want yourself and Continuum listed at the minimum, but would welcome advice from someone with more experience here.

**Comment by hafen** [comment]
I manually merged this into bokeh/rbokeh:dev, which I'm using as the staging ground for improvements as we're moving toward a CRAN release.  Thanks for the PR.  The only thing I changed is I didn't include @timelyportfolio on the copyright since I haven't heard whether he wants to or is able to be included or not.  We can update in the future as things move along.

**Pull request closed without merging**